doc/source/tutorial/getting_started.ipyml

```diff
@@ -27,6 +27,17 @@ cells:
     __version__
 
 
+- markdown: |
+    <div class="alert alert-warning">
+        The tutorial is generated from Jupyter notebooks which works in "interactive" mode (like in the LArray Editor). 
+        In the "interactive" mode, there is no need to use the function print() to display the content of a variable.
+        Simply writing its name is enough.
+        The function print() is only need if the content of more than one variable has to be displayed.<br><br>
```

Comment:
I would remove this sentence (as it is not entirely true), you can display a tuple in the console:
```python
>>> a = 1; b = 2
>>> a, b
(1, 2)
```